package.json

```diff
@@ -205,6 +207,7 @@
     "blob-polyfill": "^2.0.20171115",
     "camelcase-keys-recursive": "^0.8.2",
     "downshift": "^1.22.5",
+    "electron": "^1.7.12",
```

Comment:
Why did this change package names?

Reply:
Electron-prebuilt is deprecated. The electron package now contains the most recent built version of electron with the same api as the prebuilt package.